src/raw/cgroupv2_detect.go

```diff
@@ -22,9 +22,7 @@ type cgroupV2Paths struct {
 }
 
 func getSingleFileUintStat(cGroupV2Paths *cgroupV2Paths, stat string) (uint64, error) {
-	// get full path
-	// /sys/fs/cgroup/system.slice/cpu.weight
-	fp := filepath.Join(cGroupV2Paths.MountPoint, cGroupV2Paths.Group, stat)
+	fp := filepath.Join(cGroupV2Paths.MountPoint, cGroupV2Paths.Group)
```

Comment:
Nit: I think keeping just one `filepath.Join` is a bit more readable. WDYT?

```go
fp := filepath.Join(cGroupV2Paths.MountPoint, cGroupV2Paths.Group, stat)
c, err := ParseStatFileContentUint64(fp)
```

Reply:
Sure, I am uploading these changes after we merge https://github.com/newrelic/nri-docker/pull/105